tests/scripts/github_tvmbot.py

```diff
@@ -417,11 +433,10 @@ def author(self) -> str:
         return self.raw["author"]["login"]
 
     def find_failed_ci_jobs(self) -> List[CIJob]:
-        # NEUTRAL is GitHub Action's way of saying cancelled
         return [
             job
             for job in self.ci_jobs()
-            if job["status"] not in {"SUCCESS", "SUCCESSFUL", "SKIPPED"}
+            if job["status"] not in {"SUCCESS", "SUCCESSFUL", "SKIPPED", "NEUTRAL"}
```

Comment:
Do we want cancelled jobs (`NEUTRAL`) to be considered a success?

Reply:
ah good point, I think for now the best way would be to ignore jobs that aren't prefixed with `CI /` (since those are all the ones that don't matter for merge status)